common/v2/features/DeFiZap/components/ZapCard.tsx

```diff
@@ -99,54 +118,138 @@ const ZapCardImgSection = styled('div')`
+        <ZapCardRiskProfile>
+          {translateRaw('ZAP_RISK_PROFILE_HEADER_TEXT', {
+            $riskProfile: fetchZapRiskObject(config.risk).text
+          })}
+        </ZapCardRiskProfile>
       </ZapCardHeader>
       <ZapCardContent>
-        <ZapCardContentHeaderRow>
-          <ZapCardImgSection>
-            <img src={'https://via.placeholder.com/52'} />
-          </ZapCardImgSection>
-          <ZapCardHeaderTextSection>
-            <ZapCardHeaderTitle>{config.title}</ZapCardHeaderTitle>
-            <ZapCardHeaderName>{config.name}</ZapCardHeaderName>
-            <IndicatorItem />
-          </ZapCardHeaderTextSection>
-        </ZapCardContentHeaderRow>
-        <ZapCardContentRow>
-          <p>{config.description}</p>
-        </ZapCardContentRow>
+        {!humanReadableZapBalance ? (
+          <>
+            <ZapCardContentHeaderRow>
+              <ZapCardImgSection>
+                <Tooltip tooltip={config.breakdownTooltip}>
+                  <ZapCardImg src={config.breakdownImage} />
+                </Tooltip>
+              </ZapCardImgSection>
+              <ZapCardHeaderTextSection>
+                <ZapCardHeaderTitle>{config.title}</ZapCardHeaderTitle>
+                <ZapCardHeaderName>{config.name}</ZapCardHeaderName>
+              </ZapCardHeaderTextSection>
+            </ZapCardContentHeaderRow>
+            <ZapCardContentRow>
+              <IndicatorItem />
+            </ZapCardContentRow>
+            <ZapCardContentRow>
+              <ZapCardContentText>{config.description}</ZapCardContentText>
+            </ZapCardContentRow>
+          </>
+        ) : (
+          <>
+            <ZapCardContentHeaderRow>
+              <ZapCardImgSection>
+                <Tooltip tooltip={config.breakdownTooltip}>
+                  <ZapCardImg src={config.breakdownImage} />
+                </Tooltip>
+              </ZapCardImgSection>
+              <ZapCardHeaderTextSection>
+                <ZapCardHeaderTitle>{config.title}</ZapCardHeaderTitle>
+                <ZapCardHeaderName>{config.name}</ZapCardHeaderName>
+              </ZapCardHeaderTextSection>
+            </ZapCardContentHeaderRow>
+            <ZapCardContentRow>
+              <IndicatorItem />
+            </ZapCardContentRow>
+            <ZapCardContentRow>
+              <ZapCardContentText>{translateRaw('ZAP_BALANCE_DETECTED')}</ZapCardContentText>
+              <ZapCardContentText>
+                <ZapEstimatedBalance>
+                  {translateRaw('ZAP_ESTIMATED_BALANCE')}{' '}
+                  <Tooltip
+                    tooltip={translateRaw('ZAP_BALANCE_TOOLTIP', {
+                      $protocol: config.platformsUsed[0]
+                    })}
+                  />
+                  {`: `}
+                </ZapEstimatedBalance>
+                {`${humanReadableZapBalance.toFixed(4)} ${userZapBalances.ticker}`}
+              </ZapCardContentText>
+            </ZapCardContentRow>
+          </>
+        )}
```

Comment:
This could maybe be extracted into its own file ? it seems a bit bulky like that

Reply:
I don't think that needs to be done, but i did simplify it a bit in [47faec8](https://github.com/MyCryptoHQ/MyCrypto/pull/3094/commits/47faec884d3f17a842a69496dc2e529ba294f2df)